README.md

```diff
@@ -11,16 +11,40 @@ Senior thesis project - interactive storyteller app
 
 
 ### Server setup
-To run the backend:
-  npm run dev
+
+Run the following commands to create the database and generate the prisma client:
+```
+    To generate prisma database:
+      npx prisma generate
+      npx prisma migrate dev --name magicink
+
+    To visualise the database:
+      npx prisma studio -p 5553
+      (the android emulator and prisma studio both use port 5555 by default)
+
+    To seed a default user while authentication is currently mocked, and we default to this (getUser(1) in loginScreen):
+      npm run seed
+
+    To run the backend:
+      npm run dev
+
 
 create a `server/.env` file with the following content:
 ```
 GEMINI_API_KEY=your_gemini_api_key
 DATABASE_URL="postgresql://USER:PASSWORD@HOST:PORT/magicink?schema=public"
 ```
-Run the following commands to create the database and generate the prisma client:
-```
 
-npx prisma generate
-npx prisma migrate dev --name magicink
+create a `client/.env` file with the following content:
```

Comment:
ive added this in my read me - i changed the  the naming of the buckets to something clearer because you need to. So use mine i think